query/src/storages/fuse/operations/commit.rs

```diff
@@ -90,7 +90,7 @@ impl FuseTable {
                     match backoff.next_backoff() {
                         Some(d) => {
                             let name = tbl.table_info.name.clone();
-                            tracing::error!(
+                            tracing::warn!(
```

Comment:
@dantengsky Is it a good idea to make this log a `warning` instead of `error`?

Reply:
sounds good to me